src/App.js

```diff
@@ -14,11 +14,74 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedGoods: [],
+  }
+
+  selectBtnClick = (arr, good) => {
+    arr.push(good);
```

Comment:
```suggestion
    arr.push(good);
```
push - мутуючий метод. Ми не мутуємо стейт. Використовуй спред-оператор для додавання нового елементу в масив